transformer_engine/pytorch/module.py

```diff
@@ -452,6 +449,7 @@ def prepare_forward(
             # Activation recomputation is used and this is the first forward phase.
             if (
                 self.fp8
+                and self.training
```

Comment:
This check will omit copying the tensors for eval/inference case, which was the cause of increasing checkpoint size in the first place.